plugin.go

```diff
@@ -214,10 +220,10 @@ func (c *Plugin) GetMetadata() plugin.PluginMetadata {
 				HelpText: "Encrypt a string using a Spring Cloud Services configuration server",
 				Alias:    "csev",
 				UsageDetails: plugin.Usage{
-					Usage: `   cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME VALUE_TO_ENCRYPT
+					Usage: `   cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME [VALUE_TO_ENCRYPT]
 
-Note: if VALUE_TO_ENCRYPT contains characters that are special to the shell, it may be necessary
-to use escape characters or quotes. Refer to the shell's man page for details.`,
+      NOTE: Either VALUE_TO_ENCRYPT or --file-to-encrypt flag is required, but not both.`,
```

Comment:
Words missing from the end of this sentence? E.g. "... but not both must be specified."

Reply:
No...that seems like a complete sentence, that describes what's required. I suppose that I could expand on it a little, but I was trying to remain brief since (speaking from my own experience reading such help info/docs) I wouldn't read anything longer anyway.

Reply:
I shouldn't quibble, but the English just seems wrong to me. How about "Either VALUE_TO_ENCRYPT or --file-to-encrypt flag must be provided, but not both."? If you can't see the improvement, let's agree to differ. :-)

Reply:
@fifthposition's suggestion is great as it avoids a passive construction.